ReactCommon/yoga/yoga/YGNode.cpp

```diff
@@ -553,12 +553,12 @@ void YGNode::cloneChildrenIfNeeded() {
   }
 }
 
-void YGNode::markDirtyAndPropogate() {
+void YGNode::markDirtyAndPropagate() {
```

Comment:
The changes in the yoga folder need to be made to https://github.com/facebook/yoga then they are synced here.

Reply:
@jacobrosenthal in my prev PR it was done automatically https://github.com/facebook/react-native/pull/17578

Reply:
also Yoga has been changed, why not do the same in this PR?

Reply:
If the changes were applied to the yoga repo they’ll eventually make their way to RN sometime.

Reply:
@ide 🤦‍♂️